trio/_dtls.py

```diff
@@ -995,10 +1054,11 @@ def read_volley():
                         # PMTU estimate is wrong? Let's try dropping it to the minimum
                         # and hope that helps.
                         self._handshake_mtu = min(
-                            self._handshake_mtu, worst_case_mtu(self.endpoint.socket)
+                            self._handshake_mtu,
+                            worst_case_mtu(self.endpoint.socket),
```

Comment:
Did you accidentally add a trailing comma here which made `black` want to reformat like this?

Reply:
no clue! But reverted it